cvmfs/quota_posix.cc

```diff
@@ -368,7 +385,16 @@ PosixQuotaManager *PosixQuotaManager::CreateShared(
     LogCvmfs(kLogQuota, kLogDebug, "failed to start cache manager");
     return NULL;
   }
+  LogCvmfs(kLogQuota, kLogDebug, "new cache manager pid: %d", new_cachemgr_pid );
+  quota_mgr->SetCacheMgrPid(new_cachemgr_pid);
+  const int fd_lockfile_rw = open((workspace_dir + "/lock_cachemgr").c_str(), O_RDWR | O_TRUNC, 0600);
+  int result = SafeWrite(fd_lockfile_rw, &new_cachemgr_pid, sizeof(new_cachemgr_pid));
+  if (result == -1) {
+    LogCvmfs(kLogQuota, kLogDebug | kLogSyslogErr,
```

Comment:
I think we should also `PANIC()` because it potentially leaves other clients in an undefined state.